src/test/ui/reify-intrinsic.stderr

```diff
@@ -8,9 +8,9 @@ LL |     let _: unsafe extern "rust-intrinsic" fn(isize) -> usize = std::mem::tr
    |                                                                help: use parentheses to call this function: `std::mem::transmute(...)`
    |
    = note: expected type `unsafe extern "rust-intrinsic" fn(isize) -> usize`
-              found type `unsafe extern "rust-intrinsic" fn(_) -> _ {std::mem::transmute::<_, _>}`
+              found type `unsafe extern "rust-intrinsic" fn(_) -> _ {std::intrinsics::transmute::<_, _>}`
 
-error[E0606]: casting `unsafe extern "rust-intrinsic" fn(_) -> _ {std::mem::transmute::<_, _>}` as `unsafe extern "rust-intrinsic" fn(isize) -> usize` is invalid
+error[E0606]: casting `unsafe extern "rust-intrinsic" fn(_) -> _ {std::intrinsics::transmute::<_, _>}` as `unsafe extern "rust-intrinsic" fn(isize) -> usize` is invalid
```

Comment:
It is.. quite concerning that this PR changed this. I think I recall you asking on Discord or Zulip about this -- it'd be good to try and not have this change, since it might point at something bad going on that I've missed in my review.

Reply:
I experienced this issue on another PR (trait upcasting I think?) recently, so it seems unrelated to my changes. But then it got fixed... maybe? We'll see with the next push.

Reply:
I think it's not present in the overall PR but perhaps is in the cosmetic commit?

Reply:
Maybe. Let's see if it disappears on next push anyway (i.e., I can revert this and similar changes).

Reply:
I removed most cosmetic changes and this issue is gone now... may be unrelated actually, but I'll mark this as resolved anyway!